config/database.yml

```diff
@@ -17,13 +17,17 @@
 default: &default
   adapter: postgresql
   encoding: unicode
+  username: <%= Rails.application.credentials.user[:name]%>
+  password: <%= Rails.application.credentials.user[:password]%>
```

Comment:
hi @rloterh 

Thanks for reviewing my project kindly fill up your username and Password for The respective places if you are using Windows or for Linux remove those two lines from all places